### DISCUSSION TITLE
cocoapods vs swift package

### DISCUSSION THREAD

**Discussion by halyph** [glushchenko/fsnotes#1367] [Q&A] [answered]
_Disclaimer_: I am not MacOS/iOS developer, so this is a nube question.

FSNotes is using [cocoapods](https://github.com/glushchenko/fsnotes/wiki#how-to-compile-app-from-sources) for package management. I've checked that swift has it's own [package manager](https://www.swift.org/getting-started/#using-the-package-manager).

Could you please clarify why we don't use swift package manager? When/why swift project should use cocoapods?

**Comment by glushchenko** [comment, marked as answer]
Because cocoapods was first.

**Reply by halyph** [reply]
@glushchenko just for my understanding, would you use swift package manager if you start today?

**Reply by glushchenko** [reply]
I have no experience with Swift package manager, but why not?
